.github/workflows/citus-package-all-platforms-beta-test.yml

```diff
@@ -1,53 +0,0 @@
-name: Citus package all platforms beta tests
```

Comment:
can you share the failing test run that makes us removing this file ?

Reply:
https://github.com/citusdata/tools/actions/runs/3262600168/jobs/5359944687#step:6:159